hail/python/requirements.txt

```diff
@@ -10,5 +10,6 @@ pandas>0.22,<0.24
 parsimonious<0.9
 PyJWT
 pyspark>=2.4,<2.4.2
+python-json-logger
```

Comment:
I actually think this should be in the dev-requirements.txt, since it's not really necessary for a user, right?